stonesoup/resampler/particle.py

```diff
@@ -28,7 +28,7 @@ def resample(self, particles):
         weight = Probability(1/n_particles)
         log_weights = np.array([weight.log_value for weight in particles.weight])
         weight_order = np.argsort(log_weights, kind='stable')
-        cdf = [v.log_value for v in np.cumsum(particles.weight[weight_order])]
+        cdf = np.log(np.cumsum(np.exp(log_weights[weight_order])))
```

Comment:
@sdhiscocks @oharrald-Dstl  I changed this line. I don't think it should cause any problems, as we take the log value from the probability object to begin with, but it makes it about 10% faster.